### PR TITLE
Styling changes for references

### PR DESCRIPTION
**Pull request by discodavey**
This is for article atom

### CODE REVIEW

File: source/_patterns/00-atoms/references/reference.mustache [hunk 1]
```diff
@@ -43,9 +42,9 @@
   {{#hasAbstracts}}
     <ul class="reference__abstracts">
   {{/hasAbstracts}}
-  {{#abstracts}}
-    <li class="reference__abstract"><a href="{{url}}" class="reference__abstract_link">{{{name}}}</a></li>
-  {{/abstracts}}
+    {{#abstracts}}
```
Comment (nlisgo) [inline source/_patterns/00-atoms/references/reference.mustache:45]: Remove 2 spaces of indentation on this block

**Review by nlisgo — APPROVED** [review]

**Comment by discodavey** [comment]
@nlisgo Can you review the remaining changes for this please?

File: source/_patterns/00-atoms/references/reference.mustache (hunk 1 above)
Comment (nlisgo) [reply source/_patterns/00-atoms/references/reference.mustache:45]: Remove this indentation.

File: source/_patterns/00-atoms/references/reference~journal-article.json [hunk 2]
```diff
@@ -32,5 +112,17 @@
   "doi": {
     "doi": "10.1038/365833a0"
   },
-  "hasAbstracts": false
+
```
Comment (nlisgo) [inline source/_patterns/00-atoms/references/reference~journal-article.json:115]: nitpick: remove blank lines in json.

**Review by nlisgo — APPROVED** [review]

**Comment by nlisgo** [comment]
Are we ready to merge this in?

**Comment by discodavey** [comment]
Yes

**Pull request merged**